articles/storage/files/storage-how-to-use-files-linux.md

```diff
@@ -35,7 +35,7 @@ ms.subservice: files
 | Ubuntu Server | 14.04+ | 16.04+ |
 | RHEL | 7+ | 7.5+ |
 | CentOS | 7+ |  7.5+ |
-| Debian | 8+ |   |
+| Debian | 8+ | 9+  |
```

Comment:
@MaxMelcher Thanks for helping to improve our docs, we appreciate it! In this case there's a rather specific version of Debian 9 with a backported kernal that works. We need to specify that rather than just 9. If you would change this to "9 with backport kernal" then we could accept this change. :)

Reply:
added a note for that - thanks!